### PR TITLE
feature/937-create-refresh-token-mutation

### PR DESCRIPTION
**Pull request by bl0rch1d**
https://garage.atlassian.net/secure/RapidBoard.jspa?rapidView=36&projectKey=PEO&modal=detail&selectedIssue=PEO-937

### CODE REVIEW

File: config/locales/graphql/types.en.yml [hunk 1]
```diff
@@ -20,3 +20,9 @@ en:
           csrf: CSRF protection token
           access: Access JWT token
           refresh: JWT token for refreshing access token
+      refreshed_access_payload_type:
+        desc: Payload with refreshed access tokens
+        fields:
+          csrf: Token for signed cookies
+          access: JWT access token
+          access_expires_at: Expiration of access tokens
```
Comment (yaorlov) [inline config/locales/graphql/types.en.yml]: new line
please check your overcommit setup

File: config/credentials/development.yml.enc [hunk 2]
```diff
@@ -1 +1 @@
-9ETrt8r9RaWR3S7PRwf/LfAI/o341serTWjWbYT+y0JVLAeeJOPdBTfMvpmEPBXwSqBYWcRL3P9JrgXmPm5xpcKs2IBc4A0e82EJfxmWtv0083o7AkkOWPma8N6PrCWeuMsK7vy32839TJt68VWcygDKG+A32n2WEb3OshkxRaUEEAjruCwpLxxqTHSB+5Mlg6Sm74+/aGb9KPnnp0kbGI/P7KySxQ==--25w9e8fMXzw8jNFn--Pf4NjTCZU8yotvhSLKwwXw==
\ No newline at end of file
+/4FDrcYFj5CfplPaaT1VnRlWmagYoUvkAoHzzX0ouU6Qu7nqdmbdNhrn0p8fii3ZasEBKa9PiBluZIMQBfwoSnLb7DJZEx0qrkeDZd2QLw1GtALuHGljzSBu9+f/RDctmU3W0wULJE7O8LrRWw0FoI+ECPBZv7OQMhBqvhu5v7wtIUKH+UHgp1DSBcjzae5HAOHHUreGlvsTF9KmUo5sS1oIF7bxNigfK7rt9+FzdGPmhYkT6vSC/pBthzV/FGeZZ1AYVvnrOPx0W99Io7S/6R+8s6efJ8GqSoaHB8pZai7Pis/0VeznPTx8U6r6Ks6BZoyUSvHN8xdeag==--ytKQNRr+p8wP2YaT--34cyn3IfWZ/xRiZZvRNhuQ==
```
Comment (yaorlov) [inline config/credentials/development.yml.enc]: why did the credentials change?

Comment (bl0rch1d) [reply config/credentials/development.yml.enc]: jwt secret have been added

Comment (yaorlov) [reply config/credentials/development.yml.enc]: ok

**Pull request merged**